src/components/Web3Status/index.tsx

```diff
@@ -34,12 +34,14 @@ import Loader from '../Loader'
 import { RowBetween } from '../Row'
 import WalletModal from '../WalletModal'
 
+const FULL_BORDER_RADIUS = 9999
+
 const Web3StatusGeneric = styled(ButtonSecondary)`
   ${({ theme }) => theme.flexRowNoWrap}
   width: 100%;
   align-items: center;
   padding: 0.5rem;
-  border-radius: 14px;
+  border-radius: ${FULL_BORDER_RADIUS}px;
```

Comment:
Why can't this just be 100%?

Reply:
then it becomes an oval

Reply:
<img width="150" alt="Screen Shot 2022-09-20 at 6 18 46 PM" src="https://user-images.githubusercontent.com/4899429/191374896-976487a7-07e6-4261-bc11-44aaaa936479.png">

Reply:
Can you link to the SO post above the const?